tests/grid/tria_signals_01.output

```diff
@@ -0,0 +1,61 @@
+
+DEAL:1d-1d::n_signal_create : 1
+DEAL:1d-1d::n_signal_pre_refinement : 3
+DEAL:1d-1d::n_signal_post_refinement : 3
+DEAL:1d-1d::n_signal_pre_coarsening_on_cell : 2
+DEAL:1d-1d::n_signal_post_refinement_on_cell : 3
+DEAL:1d-1d::n_signal_copy : 1
+DEAL:1d-1d::n_signal_clear : 1
+DEAL:1d-1d::n_signal_any_change : 5
```

Comment:
> And because we can't insert signals into p4est, the signals on cell for p::d::Tria is not totally reliable as we know the p4est balancing is out of our control.

After thought twice, now I believe the cell level signals from `p::d::Tria` could cover all cell changes, because at last  `p::d::Tria`  has to know all the changes no matter what p4est has done, and the triangulation update is done by `Tria` and signals from `Tria` are verified. However my knowledge on `p::d::Tria` is very limited, I'd like to know how does @tjhei think on this problem.